tests/utils.py

```diff
@@ -591,6 +591,8 @@ def run(self, result=None):
             '-5',
             '-3',
             '5',
+            '1',
+            '-1'
```

Comment:
Is the reason for this to check if the list contains 1 and -1? I'm just wary of adding more data samples unless there's a specific reason why they're needed, as there's a performance overhead to adding more examples.